easybuild/easyconfigs/r/R/R-3.5.1-foss-2018b-X11-20180604.eb

```diff
@@ -0,0 +1,1868 @@
+#
+name = 'R'
+version = '3.5.1'
+x11ver = '20180604'
+versionsuffix = '-X11-%s' % x11ver
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://cloud.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['0463bff5eea0f3d93fa071f79c18d0993878fd4f2e18ae6cf22c1639d11457ed']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+dependencies = [
+    ('X11', x11ver),
+    ('Mesa', '18.1.1'),
+    ('libGLU', '9.0.0'),
+    ('cairo', '1.14.12'),
+    ('libreadline', '7.0'),
+    ('ncurses', '6.1'),
+    ('bzip2', '1.0.6'),
+    ('XZ', '5.2.4'),
+    ('zlib', '1.2.11'),
+    ('SQLite', '3.24.0'),
+    ('PCRE', '8.41'),
+    ('libpng', '1.6.34'), # for plotting in R
+    ('libjpeg-turbo', '2.0.0'), # for plottting in R
```

Comment:
@fizwit Two spaces are expected before the `#` according to the PEP8 style guide, see also the output of `--check-style`:

```
eb --from-pr 6726 --check-style
== temporary log file in case of crash /tmp/eb-auEg2a/easybuild-w8I2Z2.log

*** WARNING: Using easyconfigs from unstable (pending/failed tests or merge conflict) PR #6726 ***

Running style check on 1 easyconfig(s)...
/tmp/eb-auEg2a/files_pr6726/r/R/R-3.5.1-foss-2018b-X11-20180604.eb:31:26: E261 at least two spaces before inline comment
/tmp/eb-auEg2a/files_pr6726/r/R/R-3.5.1-foss-2018b-X11-20180604.eb:32:32: E261 at least two spaces before inline comment
[FAIL] /tmp/eb-auEg2a/files_pr6726/r/R/R-3.5.1-foss-2018b-X11-20180604.eb
ERROR: One or more style checks FAILED!
```